source/development_process/index.rst

```diff
@@ -0,0 +1,177 @@
+- internally determine code sprints
+
+Setting Up the Project
+----------------------
+
+- Create the git repo
+- Create the issue tracker
+- Create the project skeleton with Django
+- Create the Dockerfile
+
+The Development Process
+-----------------------
+
+- Pick an issue
+- Write tests (with a sample Django test)
+- Write the code
+- Make a PR
+- Merge it
+- Close the issue
+
+A Brief Introduction to Django
+------------------------------
+- Sample manage.py commands (runserver, collectstatic, migrate)
+- How to write a very simple model
+- How to write a very simple view
+
+How to git out of a tight corner
+--------------------------------
+- Common git configuration tricks
+- When and when not to force push
+- How to rebase code
+- How to cherry-pick
+- How to reflog
+- How to add remotes and set upstream branches
+- 
+Common Python Errors and How to Fix Them
+----------------------------------------
+- I can't think of any...
+ 
+Writing the Chef Cookbook
+-------------------------
+
+- Pretty much the same as the rest of the project
+- Setting up test kitchen (link to updated internal chef docs)
+- Create the git repo
+- Create the issue tracker
+- Create the project skeleton with Berks
+
+Writing Chef Tests
+------------------
+
+- Serverspec
+- A test cookbook
+- A Rakefile?
+- Bats?
+
+How to Write a Recipe
+---------------------
+
+- How to add dependencies
+- How to use a LWRP
+
+How to Write a Light Weight Resource Provider
+---------------------------------------------
+
+- What is an LWRP and how are they useful
+- Write a resource
+- Write a provider
+
+Common Chef Errors and How to Fix Them
+--------------------------------------
+
+Google Summer of Code
+---------------------
+The OSL has been part of the `GSoC 
+<https://www.google-melange.com/gsoc/homepage/google/gsoc2015>`_ since 2006.  
+This program has linked thousands of students from around the world with hundreds 
+of open source projects resulting in millions of lines of additional code.  
+Just about every development project at the OSL is available for students around 
+the world to join, as long as qualified mentors are available for the duration 
+of the program.  Any devs contributing to projects that are also being worked 
+on by a GSoC student should keep the following things in mind:
+
+- The student is new.
+  Their skillsets may not be as strong as other devs in areas like source 
+  control, documentation, IRC etiquette or deployment.  Patience and 
+  understanding will be greatly appreciated.  Help them become the kind of
+  developer and team member the OSL would want to hire.
+- Coordinate big changes with mentors.
+  If the project has significant changes on the horizon, fellow devs should 
+  keep the mentor up-to-date on those changes to keep from blindsiding the 
+  student.  It is hard enough for the student to learn a new workflow; 
+  minimizing "surprises" is just common courtesy.
+
+.. toctree::
+    :maxdepth: 1
+    :glob:
+
+    *
```

Comment:
Are you planning on having other docs underneath this? If not its going to create a warning and fail on the jenkins build because there's no documents there.